File: problem1/solution.rb
def sum_of_multiples_in_range(range, mult1, mult2, ints=[])
  range.first.upto(range.last).each do |i|
    ints.push(i) if i % mult1 == 0 || i % mult2 == 0
  end
  puts ints.inject(0) { |sum, value| sum += value }
end

sum_of_multiples_in_range(0..999, 3, 5)

File: problem1/solution.py
def sum_of_multiples_in_range(rng, mult1, mult2, ints=[]):
  for i in range(rng):
    if i % mult1 == 0 or i % mult2 == 0:
      ints.append(i)
  print sum(ints) 

sum_of_multiples_in_range(1000, 3, 5)
